containers/datalab/build.sh

```diff
@@ -25,13 +25,21 @@ VERSION_SUBSTITUTION="s/_version_/0.5.$VERSION/"
 COMMIT=`git log --pretty=format:'%H' -n 1`
 COMMIT_SUBSTITUTION="s/_commit_/$COMMIT/"
 
+if [ -z "$1" ]; then
```

Comment:
Why do we need this change? `pydatalabPath` isn't used anywhere in this script, other than being passed without modification to `base/build.sh`, which takes care of getting its `realpath`.